docs/source/actionchain.rst

```diff
@@ -21,11 +21,11 @@ Following is sample ActionChain workflow definition named :github_st2:`echochain
 Details:
 
 * ``chain`` is the array property that contains tasks, which incapsulate action invocation.
-* Tasks are named action execution specifications. The name is scoped to an ActionChain and is used as a reference to a task.
+* Tasks are named action execution specifications, seperated by ``-``. The name is scoped to an ActionChain and is used as a reference to a task.
```

Comment:
Ah! Something like 

"Tasks are named action execution specifications provided in a list."